### PR TITLE
docs: clarify API routing (and with --web.external-url)

### PR DESCRIPTION
**Pull request by adamdecaf**
I couldn't find a reference for determining API routes on the readme, after some trial and error I found out what worked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,15 +180,17 @@ receivers:
...
 of the HTTP handlers themselves. The API specification can be found in
 [api/v2/openapi.yaml](api/v2/openapi.yaml). A HTML rendered version can be
-accessed
-[here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
-Clients can be easily generated via any OpenAPI generator for all major
-languages.
+accessed [here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
+Clients can be easily generated via any OpenAPI generator for all major languages.
+
+Note: Under the default config endpoints are accessed under a `/api/v1` or `/api/v2` prefix.
```
Comment (stuartnelson3) [inline README.md]: May I suggest a small edit?

"With the default config, endpoints are accessed..."

File: README.md [hunk 2]
```diff
@@ -180,15 +180,17 @@ receivers:
...
+The v2 `/status` endpoint would be `/api/v2/status`. If `--web.external-url` is set then API routes are
+prefixed with that as well, so `--web.external-url=https://int.company.com/alertmanager/` would
```
Comment (brian-brazil) [inline README.md]: This isn't fully correct, it's --web.route-prefix that matters.

**Comment by adamdecaf** [comment]
Thanks. I've updated after your suggestions.

File: README.md [hunk 3]
```diff
@@ -180,15 +180,17 @@ receivers:
...
-accessed
-[here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
-Clients can be easily generated via any OpenAPI generator for all major
-languages.
+accessed [here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/prometheus/alertmanager/master/api/v2/openapi.yaml).
+Clients can be easily generated via any OpenAPI generator for all major languages.
+
+With the default config, endpoints are accessed under a `/api/v1` or `/api/v2` prefix.
+The v2 `/status` endpoint would be `/api/v2/status`. If `--web.route-prefix` is set then API routes are
+prefixed with that as well, so `--web.route-prefix=https://int.company.com/alertmanager/` would
```
Comment (brian-brazil) [inline README.md]: It'd just be /alertmanager/ for this flag.

**Comment by adamdecaf** [comment]
Fixed and rebased.

**Pull request merged**